packages/salesforcedx-vscode-core/src/commands/util/postCheckers.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
```

Comment:
I've added a new `util` submodule in the `commands` module. In this PR it houses the globStrategies, sourcePathStrategies, and new location for the FilePathExistsChecker. `commands.ts` is becoming a rather large mixed bag, so I thought we could lay some groundwork for later to break it up a bit into more readable files. Perhaps something to do during our time allocation for this kind of work.

Let me know what you think of going this route, I am open to whatever.

Reply:
I think this is a good idea! I was going to ask whether we should include it in the utils folder directly in the `core` package, but I think this structure makes more sense because we don't need these outside the `commands` scope.